wbooks/src/app/components/CommentList/index.tsx

```diff
@@ -9,26 +9,23 @@ import styles from './styles';
 
 const CommentList = ({ comments }: CommentsData) => {
   const allCommentsAmount = comments.length;
-  const [initialAmount, setNumber] = useState(2);
+  const [commentsToView, setNumber] = useState(2);
   const viewAll = () => {
     setNumber(allCommentsAmount);
   };
 
-  const renderItem = ({ item }: { item: CommentsProps }) => (
-    <Comment key={item.id} name={item.name} img={item.img} comment={item.comment} />
-  );
-  const renderSeparator = () => <View style={styles.cardSeparator} />;
+  const renderItem = ({ id, name, img, comment }: CommentsProps, index: number) =>
+    index < commentsToView ? (
+      <View key={id}>
+        <Comment key={id} name={name} img={img} comment={comment} />
+        <View style={styles.cardSeparator} />
+      </View>
+    ) : null;
 
   return (
     <View>
-      <FlatList
-        data={comments}
-        renderItem={renderItem}
-        ItemSeparatorComponent={renderSeparator}
-        initialNumToRender={initialAmount}
-        windowSize={1}
-      />
-      {initialAmount === 2 && (
+      {comments.map(renderItem)}
```

Comment:
```suggestion
      {comments.slice(0, commentsToView - 1).map(renderItem)}
```
and remove the `index < commentsToView ? ` above